www/js/app.js

```diff
@@ -1151,7 +1156,7 @@ define(['jquery', 'zimArchiveLoader', 'uiUtil', 'settingsStore','abstractFilesys
             if (lastSelectedArchive !== null && lastSelectedArchive !== undefined && lastSelectedArchive !== '') {
                 // Attempt to select the corresponding item in the list, if it exists
                 if ($("#archiveList option[value='"+lastSelectedArchive+"']").length > 0) {
```

Comment:
Will you be converting this in a different PR?

Reply:
I wasn't sure about this one. I will do it in a different PR.